pkg/apis/k0s.k0sproject.io/v1beta1/network.go

```diff
@@ -38,17 +38,20 @@ type Network struct {
 	Provider string `json:"provider"`
 	// Network CIDR to use for cluster VIP services
 	ServiceCIDR string `json:"serviceCIDR,omitempty"`
+	// Cluster Domain
+	ClusterDomain string `json:"clusterDomain,omitempty"`
```

Comment:
I'm not sure about the `omitempty` here. A default is set below, but as is for Provider and ServiceCIDR where one has `omitempty` and the other not